README.md

```diff
@@ -1,2 +1,54 @@
 # xxxd
-An alternative to xxd - binary to source converter - download binaries for major OSs from the releases page.
+An alternative to xxd.  Xxd is sometimes distributed with vim, whereas this is standalone.
+
+A binary file to text converter.
+
+It is particularly useful for converting a file into a format that can be understood
+by a C compiler.
+
+See the [Releases](releases/) area to download binaries for major OSs.
+
+## Differences to xxd
+
+* Long options must be used with double dashes, not single dashes.
+* The single letter options are the same as xxd, but some long form options have been added:
+    * `--compact` A more compact output format for C format.
+    * `--indentation` Change the default indentation (2 for C format, 0 for other formats).
+    * `--little-endian` An option that changes the endianness of multi-byte words without changing the format.
+    * `--tabs` Use a mixture of tabs and spaces for indentation.
+    * `--tab-width` Change the default tab width if tabs are used (8).
+* Does not have a -r option to convert text back to binary. The `--seek` option is therefore also not supported.
+* The `-c` (columns) option is not limited to 256.
+* The `-E` (EBCDIC) option is not supported.
+* The `-p` (postscript) option has only one long form option, `--postscript`.  The `--ps` and `--plain` aliases are not supported.
+* The `-s` option does not accept unary plus, and negative seeks are not supported.
+
+# Usage
+
+```
+Usage:
+  xxxd.toit [<options>] [--] [<in:file>] [<out:file>]
```

Comment:
I imagine we will ship this as precompiled binaries so it probably should just be `xxxd`.